packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -134,6 +133,18 @@ await createMyThenable();
 This option, which is `true` by default, allows you to stop the rule reporting promises consumed with the [`void` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void).
 This can be a good way to explicitly mark a promise as intentionally not awaited.
 
+:::warning
+Voiding a Promise doesn't handle it or change the runtime behavior. The outcome is just ignored, like disabling the rule with a
+
+```
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+```
+
+Such Promise rejection will still lead to an [`unhandledrejection` event](https://developer.mozilla.org/en-US/docs/Web/API/Window/unhandledrejection_event)
```

Comment:
Continuing from https://github.com/typescript-eslint/typescript-eslint/pull/9949#discussion_r1748907511,

> ... my idea was to exactly represent what happend to me and what I described in the issue (the code example).

Right, exactly. But the details of exactly what happened to you are a little specific imo since it's dependent on node vs web, and on whether a global promise rejection handler was or was not set up.

> So, I think it is important to include a link to mdn page about `unhandledrejection`, which can explain surely better what is it.
> 
> About the comment above, I think could be interesting to give a hint about how can a person handle a promise.

To be clear, I definitely agree with putting this info on this page! 🙂  I'm just thinking it isn't _specific_ to the `void` operator - it's essential knowledge whether or not you're using the `ignoreVoid` option - so I'd prefer it to be in the general info on handling promises, rather than in this particular warning block (which I wouldn't bother reading if i had the `ignoreVoid` option disabled).

Note that the MDN page link (albeit different section) is included already, down in the "Further Reading" section. Maybe we can find a better place to link the "error handling" section in the intro?

What do you think?

Reply:
> Right, exactly. But the details of exactly what happened to you are a little specific imo since it's dependent on node vs web, and on whether a global promise rejection handler was or was not set up.

I just tested the difference between node and web. I see some differences I'm going to document here.

 - `unhandledrejection` (web) vs `unhandledRejection` (node)
 - `window` vs `process`
 - if the listener is set up, error log still prints on web, but doesn't on node

So, I see your point.

> so I'd prefer it to be in the general info on handling promises, rather than in this particular warning block (which I wouldn't bother reading if i had the ignoreVoid option disabled).

I agree. The link to `ignoreVoid` was serving that exact purpose, but it might not be enough. There is no section about handling promises, right?

The fact the MDN link is on the end section might not help making it being read. Personally I don't ever read to the end of a document. This behavior might be wrong, but I organize my document sections to give the important resources first.

I'm going to try to put down a few lines